src/main/steps/home/get.ts

```diff
@@ -78,6 +79,9 @@ const applicant1RedirectPageSwitch = (caseState: State, userCase: Partial<Case>,
     case State.AwaitingGeneralConsideration: {
       return HUB_PAGE;
     }
+    case State.ConditionalOrderDrafted: {
+      return READ_THE_RESPONSE;
+    }
```

Comment:
This is only in the case for a sole application, but you could leave this in and I can update my pr after:

https://github.com/hmcts/nfdiv-frontend/pull/1531/files#diff-461aadf81468465bfe6c5fcfc5e41e1b079e65e9ce1f5d5cae039eedb291514eR79-R81